README.md

```diff
@@ -87,17 +88,20 @@ const addOns: Array<ssp.ClusterAddOn> = [
     new ssp.addons.ArgoCDAddOn,
     new ssp.addons.CalicoAddOn,
     new ssp.addons.MetricsServerAddOn,
+    new ssp.addons.ClusterAutoScalerAddOn,
     new ssp.addons.ContainerInsightsAddOn,
     new ssp.addons.AwsLoadBalancerControllerAddOn(),
+    new ssp.addons.NginxAddOn,
     new ssp.addons.VpcCniAddOn(),
     new ssp.addons.CoreDnsAddOn(),
-    new ssp.addons.KubeProxyAddOn()
+    new ssp.addons.KubeProxyAddOn(),
+    new ssp.addons.XrayAddOn()
 ];
 
-const account = '<YOUR_ACCOUNT_ID'
-const region = 'us-east-2'
+const account = 'XXXXXXXXXXXXX'
+const region = 'us-east-1'
```

Comment:
Why change the region?

Reply:
It is actually difference between README and QUICKSTART guide's sample. So thought to be consistent at both place with same code to make future maintenance easier. Is there specific region to consider?